### DISCUSSION TITLE
Negative values and int&float convertors

### DISCUSSION THREAD

**Discussion by jvail** [encode/starlette#1778] [Potential Issue] [answered]
Hi,

I may have missed the discussion/issue but I wonder what the reason might be to not support negative values in the default float & int convertors?

I have now implemented them myself (by copying starlette's default converters and changing the regex). But of course I'd prefer not to do that.

Thank you
Jan

**Comment by Kludex** [comment, marked as answer]
The decision is not documented, but I assume it was made to match what other web frameworks had by default.

Also, as you said, you can implement your own converter. If this gets more attraction, we can rethink the decision.
